changelogs/fragments/7797-ipa-fix-otp-idempotency.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ipa_otptoken - The module expect `ipatkendisabled` as string but the `ipatokendisabled` value retrun in `boolean`.
```

Comment:
```suggestion
  - ipa_otptoken - the module expect ``ipatokendisabled`` as string but the ``ipatokendisabled`` value is returned as a boolean (https://github.com/ansible-collections/community.general/pull/7795).
```

Reply:
I checked the FreeIPA project documentation and discovered that the ipatokendisabled attribute is only available in FreeIPA v4, meaning it was not present in earlier versions.

In the API documentation for FreeIPA v4 (API.txt: https://github.com/freeipa/freeipa/blob/ipa-4-0/API.txt#L2303), this option is clearly defined as a Boolean.

Given that these modules were developed just three years ago and FreeIPA v4 was released over nine years ago, I believe this was likely a typo.

Reply:
@felixfontein PTAL